docs/whats-new.asciidoc

```diff
@@ -13,140 +13,194 @@ Other versions: {security-guide-all}/8.5/whats-new.html[8.5] | {security-guide-a
+*Block hashes and malicious files*
 
-*Enhanced Cloud Posture dashboard* 
+You can now {security-guide}/indicators-of-compromise.html#add-indicator-to-blocklist[add indicator values] to the blocklist to prevent selected applications from running on your hosts. You can use MD5, SHA-1, or SHA-256 hash values from `file` type indicators.
 
-The Cloud Posture dashboard has an improved user interface, including more ways to directly access the Findings page.
+*Improvements to indicators*
 
-[role="screenshot"]
-image::whats-new/images/8.6/cloud-dashboard.png[Cloud Posture dashboard]
+There are several enhancements and UI improvements to indicators:
+
+* When you investigate an indicator in Timeline, the query's time range is set to seven days before and after the indicator's `timestamp`.
+* You can now delete values entered into the *Stack by* field in the Trend chart.
+* When you open the Indicator table's field browser, the `agent`, `base`, and `event` field categories are preselected to narrow the scope of relevant fields.
+* The Indicator details flyout now has the *Overview* and *Table* tabs when you open indicator details from a case comment.
 
 [float]
-== Threat intelligence enhancements
+== Cases enhancements
 
-*Updated Threat intelligence UI*
+*Cases can be shared*
 
-A *View indicators* button has been added to the Threat Intelligence view on the Overview dashboard. Click this button to go directly to the Indicators page, where you can view more details about ingested indicators.
+Each case has a universally unique identifier (UUID) that you can {security-guide}/cases-open-manage.html#cases-copy-case-uuid[copy and share]. You can access a case's UUID from the Cases or or case details pages.
 
 [role="screenshot"]
-image::whats-new/images/8.6/TI-card.png[Threat Intelligence card]
+image::whats-new/images/8.7/cases-copy-case-id.png[Copy Case ID option in More actions menu 40%,40%]
+
+*Enhancements to cases*
+
+Several changes have been made to the Cases table to improve usability:
+
+* Now you can bulk edit assignees.
+* The new *Updated on* column shows the last time cases were modified.
+* The *Status*, *Severity*, and *Name* columns are sortable.
+* Any filters and sorting you apply to the table persist in the browser.
 
-*Add cases to indicators*
 
-Adding indicators to cases allows you to escalate threats to other teams and workflows using {elastic-sec} cases. You can add indicators to cases from the Indicators table or the Indicators details flyout.
+[float]
+== Osquery enhancements
+
+*Event and alert data can be dynamically transferred into Osquery queries*
+
+Instead of hard-coding alert and event values into Osquery queries, you can now use {security-guide}/osquery-placeholder-fields.html[placeholder fields] to dynamically pass this data into queries. Placeholder fields function like parameters can be used to build flexible and reusable queries.
```

Comment:
```suggestion
Instead of hard-coding alert and event values into Osquery queries, you can now use {security-guide}/osquery-placeholder-fields.html[placeholder fields] to dynamically pass this data into queries. Placeholder fields function like parameters and can be used to build flexible and reusable queries.
```